lifecycle.md

```diff
@@ -48,9 +48,24 @@ Note: this content should be moved to another location as it is not lifecycle re
 
 Build and test images are referenced in repo infra files, across a variety of `main` and `release/*` branches. Updating these references is a multi-step detail-oriented task. It is a pain, but necessary.
 
-We use version specific references in infra to ensure that our CI builds are reliable. One can imagine using floating OS tags (such as `debian-oldest` and `debian-latest`), however such an approach would be guaranteed to break our build. We know that since we often see build and test breaks that need addressing in PRs where we update build and test images.
+Several types of tag styles are provided:
```

Comment:
Would these tags be created regardless if they are being used or would they be created on demand?  I question if they are going to be used for every distro/image variant.

Reply:
I also lean towards on-demand. In fact, I'm not sure we want all distros being tested with floating tags.

Reply:
> Several types

Several of two?